cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
```

Comment:
For the first line then:

    YARN containers (both :term:`master services` and user programs) launched by CDAP by default use the 
    logback configuration file present in the :ref:`Configuration <install-configuration>` directory:  
    ``logback-container.xml``.

Except it looks like standalone uses "logbook.xml".